dask_cuda/local_cuda_cluster.py

```diff
@@ -399,6 +399,12 @@ def __init__(
 
         self.pre_import = pre_import
 
+        if "resources" in worker_kwargs:
+            if "GPU" not in worker_kwargs["resources"]:
+                worker_kwargs["GPU"] = 1
+        else:
+            worker_kwargs["resources"] = {"GPU": 1}
```

Comment:
I wonder if it makes sense to use `{"cuda": 1}` (i.e. use "cuda" as the resource name for a CUDA-capable GPU)?
This would align with the convention used in DL (e.g. pytorch)

Reply:
I wouldn't mind personally, but the keyword `GPU` is documented in [Distributed worker resources docs](https://distributed.dask.org/en/stable/resources.html), so that seems like a more universal option and better documented.

Reply:
Okay nice - That's good enough for me.